packages/shared/lib/Cargo.toml

```diff
@@ -17,6 +17,9 @@ multicore = ["rayon", "wasm-bindgen-rayon", "namada_sdk/multicore"]
 nodejs = []
 web = []
 
+[build-dependencies]
+namada_tx = { git = "https://github.com/anoma/namada", tag = "v0.45.1" }
```

Comment:
We should bump this to `v0.46.0` here - this may cause issues since this was recently merged: https://github.com/anoma/namada-interface/blob/f326f42fffba8a53e0dec8b440bb98dd907ec059/packages/shared/lib/Cargo.toml#L27

Reply:
I will rebase and bump. thanks! :)